routes/api_camps_routes.js

```diff
@@ -1324,4 +1336,31 @@ module.exports = (app, passport) => {
             });
     })
 
+    // Delete, make camp inactive
+    app.post('/camps/:id/updatePreSaleQuota', userRole.isAdmin(), (req, res) => {
+        Camp.forge({ id: req.params.id })
+            .fetch().then((camp) => {
```

Comment:
Why not use async-await instead of promises?

For an example, see the implementation of this route: '/camps/:camp_id/documents/:doc_type/'